cdap-proto/src/main/java/io/cdap/cdap/proto/artifact/ChangeSummary.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2014-2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto.artifact;
+
+import io.cdap.cdap.api.annotation.Beta;
+
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * Represents the change summary for an update of the application
+ */
+@Beta
+public class ChangeSummary {
+    @Nullable
+    protected final String changeSummary;
```

Comment:
this name is redundant, it would result in an object like:

```
{
  "changeSummary": {
    "changeSummary": "...."
  }
}
```

 call it description instead.